releasenotes/releasenotes_v5.rst

```diff
@@ -0,0 +1,165 @@
+#. Empty search parameters are now ignored by the server instead of resulting in an error response.
+#. Firely Server now creates valid FHIR R5 AuditEvents.
+#. Firely Server now supports searching on version-specific references. Consult the `FHIR specification <https://www.hl7.org/fhir/search.html#versions>`_ for more information.
+#. Searching for a resource with multiple sort fields does not throw an exception anymore when Firely Server runs on a SQL database.
+#. Firely Server now generates FHIR AuditEvent resources conforming to `IHE Basic Audit Log Patterns <https://profiles.ihe.net/ITI/BALP/index.html>`_. Fields that are included in the audit event log and AuditEvent resources now contain the same content.
+#. When using the ``If-Modified-Since`` header, only resources that were modified after the specified timestamp are returned. Because of a precision mismatch (seconds vs. milliseconds), wrong resources were sometimes returned before this fix.
+#. When updating a deleted resource conditionally, Firely Server does not throw an exception anymore.
+#. Firely Server now returns the correct issue code when performing a conditional update using ``_id`` as a parameter. Additionally, the error message has been improved when a resource in a different information model is matched via the ``id`` field.
+#. When executing a ``POST``-based search, Firely Server will now return the correct self-link as seen in ``GET``-based searches.
+#. Upon commencing a Bulk Data Export, Firely Server now correctly handles ``Prefer`` headers as outlined `in the specification <https://hl7.org/fhir/uv/bulkdata/export/index.html#headers>`_.
+#. ``Device`` can now be added as an additional resource in a Bulk Data export.
+#. The client id of the default SMART authorization options have been changed from ``vonk`` to ``firelyserver``.
+#. Firely Server now returns improved error messages if the client is not allowed to perform searches.
+#. Support for Firely Server using a SQLite database on arm64-based Macs was improved. 
+#. During SMART on FHIR v2 discovery, Firely Server now returns the ``grant_types_supported`` field.
+#. Firely Server now returns the correct CodeSystem ``http://terminology.hl7.org/CodeSystem/restful-security-service`` within the security section of its ``CapabilityStatement``. Before this change, the old R3 CodeSystem ``http://hl7.org/fhir/restful-security-service`` was falsely returned.
+#. When performing a Bulk Data Export request with a Firely Server instance running on a SQL database, it will return the Group resource even if it has no members. 
+#. Firely Server will now handle duplicate DLLs and assemblies more gracefully in case they were accidently added to its plugin directory.
+
+Configuration
+^^^^^^^^^^^^^
+#. The configuration section for additional endpoints in the discovery document and additional issuers in tokens has been reworked. Consult the :ref:`SMART Configuration section<feature_accesscontrol_config>` for more details.
```

Comment:
We should add a warning here that the old config will stop working, maybe even outline what has been changed.

Reply:
Added a warning but I want to avoid duplicating information here. Otherwise we'll have to remember to change both places. Keeping a reference is better 👍